src/main/resources/config/application-artemis.yml

```
@@ -9,6 +9,7 @@ artemis:
   repo-download-clone-path: ./repos-download      # a temporary folder, in which git repos are downloaded that are immediately deleted afterwards (e.g. exports, plagiarism checks), should NOT be in a shared file system area
   data-export-path: ./data-exports                # a folder in which data exports are stored
   build-logs-path: ./build-logs                   # a folder in which build logs are stored
+  temp-path: ${java.io.tmpdir}                           # a folder in which temporary files are stored, e.g. for the online code editor
```

Can we use the same indentation level here? 

```suggestion
  temp-path: ${java.io.tmpdir}                    # a folder in which temporary files are stored, e.g. for the online code editor

```